ui/src/components/Settings/Notifications/CreateNotificationModal/index.tsx

```diff
@@ -0,0 +1,302 @@
+              </div>
+            </div>
+            {/* Enabled */}
+            <div className="form-row">
+              <label htmlFor="enabled" className="text-label">
+                Enabled
+              </label>
+              <div className="form-input">
+                <div className="form-input-field">
+                  <input
+                    type="checkbox"
+                    name="enabled"
+                    id="enabled"
+                    defaultChecked={props.selected?.enabled}
+                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                      event.target.value === 'on'
+                        ? (enabledRef.current = true)
+                        : (enabledRef.current = false)
+                    }
+                  ></input>
+                </div>
+              </div>
+            </div>
+            {/* Select agent */}
+            <div className="form-row">
+              <label htmlFor="ssl" className="text-label">
+                Agent *
+              </label>
+              <div className="form-input">
+                <div className="form-input-field">
+                  <select
+                    id="agent"
+                    name="agent"
+                    onChange={(e) => {
+                      setFormValues({})
+                      setTargetAgent(availableAgents[Number(e.target.value)])
+                    }}
+                    className="rounded-l-only"
+                  >
+                    {availableAgents?.map((agent, index) => (
+                      <option
+                        key={`agent-${index}`}
+                        value={index}
+                        selected={
+                          props.selected
+                            ? agent.name === props.selected?.agent
+                            : false
+                        }
+                      >
+                        {`${agent.name}`}
+                      </option>
+                    ))}
+                  </select>
+                </div>
+              </div>
+            </div>
+
+            <div className="form-row">
+              <label htmlFor="about-scale" className="text-label">
+                Days for About To Be Handled
+              </label>
+              <div className="form-input">
+                <div className="form-input-field">
+                  <input
+                    type="number"
+                    name="about-scale"
+                    defaultValue={
+                      props.selected?.aboutScale || aboutScaleRef.current
+                    }
+                    onChange={(event: React.ChangeEvent<HTMLInputElement>) =>
+                      (aboutScaleRef.current = +event.target.value)
+                    }
+                  ></input>
+                </div>
+              </div>
+            </div>
+
+            <div>
+              {/* Load fields */}
+              {targetAgent?.options.map((option) => {
```

Comment:
I think we should have a different input for the Json Payload option. The monaco-editor would be perfect here.

Reply:
Ah yes, good suggestion!